### ISSUE TITLE
Getting strange warning: To ignore the warning for all files run 'rvm rvmrc warning ignore allGemfiles'.

### ISSUE THREAD

**Issue by gabrielshih** [rvm/rvm#2939]
I am trying to tech myself programming (starting with RoR) and after installing Heroku I now get this message when I go into my RoR project directory:

$ cd etsydemo
RVM used your Gemfile for selecting Ruby, it is all fine - Heroku does that too,
you can ignore these warnings with 'rvm rvmrc warning ignore /Users/rgabrielshih/etsydemo/Gemfile'.
To ignore the warning for all files run 'rvm rvmrc warning ignore allGemfiles'.

As you can see, I am using Alex Yang's RoR tutorial on Udemy. The problem didn't start until after I setup Heroku. 

Can someone explain in layman's terms what is happening and why I am getting this message? Is it important? Should I just run the command to ignore the warnings?

Thanks. 
-Gabe

**Comment by cepbrian** [comment]
Wait, so rvm hacks cd so that it does something whenever I change directories?

**Comment by mpapis** [comment]
@cepbrian it depends on your definition of hacking, RVM uses a tool to do that - https://github.com/mpapis/bash_zsh_support/tree/master/chpwd - it simulates the ZSH functionality, and multiple projects use this tool, it allows any project to hook into `cd` exactly the same way as you do in ZSH
